lib/modules/commentTools.js

```diff
@@ -832,6 +832,17 @@ const magicPlaceholders = [
 			return date.toDateString();
 		},
 	}, {
+		matches: ['flair'],
+		description: 'On comments pages, returns the flair text of the original link',
+		handle() {
+			if (isPageType('comments')) { 
+				var linkflair = document.querySelector('.linkflairlabel').innerHTML;
```

Comment:
Could you use `textContent` instead of `inner HTML`?  There might be extra markup injected in there now or in the future, seems like only the text is interesting for comments.